website/docs/concepts/router.md

```diff
@@ -3,13 +3,21 @@ title: "Router"
 description: "Yew's official router"
 ---
 
-[The router on crates.io](https://crates.io/crates/yew-router)
-
 Routers in Single Page Applications (SPA) handle displaying different pages depending on what the URL is.
 Instead of the default behavior of requesting a different remote resource when a link is clicked,
 the router instead sets the URL locally to point to a valid route in your application.
 The router then detects this change and then decides what to render.
 
+Yew provides router support in the `yew-router` crate. To start using it, add the dependency to your `Cargo.toml`
+
+<--! Reminder: fix this when we release a new version of yew -->
+```toml
+yew-router = {git = "https://github.com/yewstack/yew.git"}
+```
+
+The utilities needed are provided under `yew_router::prelude`,  
+
+
```

Comment:
I find this necessary but missing - actually it exists under `packages/yew_router/readme.md`!! It's appropriate to move it here leave a reference only, as discussed in #2183